src/components/EnterpriseContract/EnterpriseContractTable/EnterpriseContractExpandedRowContent.tsx

```diff
@@ -0,0 +1,61 @@
+import {
+  DescriptionList,
+  DescriptionListDescription,
+  DescriptionListGroup,
+  DescriptionListTerm,
+} from '@patternfly/react-core';
+import { ExpandableRowContent, Tr } from '@patternfly/react-table';
+import { ExternalLink, Timestamp } from '../../../shared';
+import { UIEnterpriseContractData } from '../types';
+
+import './EnterpriceContractTable.scss';
+
+interface Props {
+  obj: UIEnterpriseContractData;
+  isExpanded: boolean;
+}
+
+export const EnterpriseContractExpandedRowContent: React.FC<Props> = ({ obj, isExpanded }) => {
+  if (!isExpanded) return null;
+  if (!obj.description && !obj.collection?.length && !obj.solution && !obj.timestamp) return null;
+
+  return (
+    <Tr className="ex-expanded-row" data-test="ec-expand-content">
+      <ExpandableRowContent>
+        <DescriptionList className="ec-description-list">
+          <DescriptionListGroup>
+            <DescriptionListTerm>Rule Description</DescriptionListTerm>
+            <DescriptionListDescription>{obj.description ?? '-'}</DescriptionListDescription>
+          </DescriptionListGroup>
+
+          {obj.collection?.length ? (
+            <DescriptionListGroup>
+              <DescriptionListTerm>Collection</DescriptionListTerm>
+              <DescriptionListDescription>
+                <ExternalLink href="https://enterprisecontract.dev/docs/ec-policies/release_policy.html#_available_rule_collections">
```

Comment:
Link gives 404 + I belive we do not hard code documentation links into files. 
![Screenshot 2025-06-04 at 11 05 27](https://github.com/user-attachments/assets/5cf40789-07ca-4737-af02-23aac3136093)

Reply:
hmm, but we did not change the link in this PR - it's the same one that was before the refactor 🤔